github-orgs/kubeflow/org.yaml

```diff
@@ -724,6 +725,7 @@ orgs:
             - k8s-ci-robot
             - kubeflow-bot
             - google-oss-robot
+            - aws-kf-ci-bot
```

Comment:
/cc @jlewi 
Why does aws-kf-ci-bot need write permissions to these repos?

I think based on discussion in the design doc, it only needs permission to update github PR statuses.
In that case, I'd suggest setting up a new team, probably named like `third-party-bots` and give the minimal permissions it needs.

Reply:
So there's only admin / write / read access to be granted, I'll make a separate team-list as 'third-party-bots' and start from write permissions to katib / pytorch-operator repo.

Reply:
Prow doesn't support bot to fine grain control the permissions yet. Let's try to make the POC work first and limit the repo it has access to.